src/monsters.test.c

```diff
@@ -202,6 +208,14 @@ static void test_dragon_moves(CuTest * tc)
     plan_monsters(f2);
 
     CuAssertPtrNotNull(tc, find_order("move east", m));
+
+    mt_register(mt_new_va("dragon_growl", "dragon:unit", "number:int", "target:region", "choice:int", 0));
+
+    random_growl(m, findregion(1, 0), 3);
+
+    msg = test_get_last_message(r->msgs);
+    assert_message(tc, msg, "dragon_growl", 4, m, 1, findregion(1,0), 3);
+
```

Comment:
Die exakten Werte in der Message zu testen, das spare ich mir im Grunde überall sonst auch, weil es zu kompliziert ist. Das die Message erzeugt wurde, ist oft gut genug (test_find_messagetype gibt es glaube ich noch, das ist etwas stabiler als test_get_last_message).